topics/galaxy-interface/tutorials/workflow-fairification/tutorial.md

```diff
@@ -0,0 +1,668 @@
+**Licence** | The standard licence is Apache Software Licence 2.0. If you did not make the workflow yourself, be sure that the licence corresponds to the licence where you took the workflow from (for example github licences). | No
+**Sharing** | Specify who can view the summary, get access to the content, and edit the Workflow. This is possibly already filled in according to the selected project. | No
+**Tags** | Choose an appropriate tag for your workflow. Please check if your tag is already available and use the existing one if so. If you make a new tag, keep it simple without capitals or spaces. For example all new covid-19 workflows need to be tagged with covid-19. | No
+**Creators** | This is an important section where all the people that were involved in making / publishing this workflow are listed. | No
+
+> <question-title></question-title>
+>
+> 1. Which WorkflowHub metadata is supported in Galaxy workflow interface?
+> 2. Which values should we put for our workflow?
+>
+> > <solution-title></solution-title>
+> >
+> > Workflow metadata | Galaxy metadata | Mapping workflow value
+> > --- | --- | ---
+> > Title | Name | Quality control and mapping
+> > Description | Annotation | Workflow runs quality control and mapping of paired-end short-reads data.
+> > Source | *Not supported in Galaxy workflow interface* | 
+> > Teams | *Not supported in Galaxy workflow interface* | 
+> > License | License | MIT License
+> > Sharing | *Not supported in Galaxy workflow interface* | 
+> > Tags | Tags | `sequence-analysis`, `mapping`
+> > Creators | Creator | 
+> >
+> {: .solution}
+{: .question}
+
+Let's annotate the workflow
+
+> <hands-on-title>Add metadata to the workflow</hands-on-title>
+>
+> 1. Add a proper title in **Name**
+> 2. Add a workflow description in **Annotation**
+> 3. Select a license in **License**
+>
+>    > <comment-title></comment-title>
+>    > The most appropriate **license** are **MIT** or **GNU Affero** (if you want a [copyleft](https://en.wikipedia.org/wiki/Copyleft)).
+>    {: .comment}
+>
+> 4. Add creators in **Creator** with their name and a unique identifier (typically an orcid.org ID)
+>
+>    > <comment-title></comment-title>
+>    > Do not forget to click on **Save**
+>    {: .comment}
+>
+> 5. Add tags in **Tags**
+> 6. Save the workflow
+{: .hands_on}
+
+Tip 2 is now fulfilled. We can now move toward the other left tips:
+
+- Tip 1 (**Findability**): *"Register the workflow."*
+- Tip 3 (**Accessibility**): *"Make source code available in a public code repository."*
+- Tip 4 (**Accessibility**): *"Provide example input data and results along with the workflow."*
+- Tip 10 (**Reusability**): *"Provide clear and concise workflow documentation."*
+
+   The workflow annotation provides a short and concise description of the workflow. So Tip 10 is partially fullfilled. In addition to the workflow annotation, each step could annotated so users could have an idea about the purpose of the steps
```

Comment:
Can we mention here the fancy new workflow annotations from Laila as well? markdown inside the workflow, color encoded separations etc ... they are Galaxy specific but are imho also enhancing the documentation.